### PR TITLE
Added Locations Listing, Added Team/Member Listing

### PR DESCRIPTION
**Pull request by ChrisChasm**
These two functions support the edits to the template-settings.php

### CODE REVIEW

File: dt-users/users-template.php [hunk 1]
```diff
@@ -257,68 +285,88 @@ function dt_build_user_fields_display( array $usermeta ): array
...
-                    'user_email'   => '',
-                    'user_url'     => '',
-                ],
-                [
-                    'ID'           => '',
-                    'display_name' => '',
-                    'user_email'   => '',
-                    'user_url'     => '',
-                ],
+function dt_get_user_team_members_list( int $user_id ): array {
```
Comment (corsacca) [inline dt-users/users-template.php]: @ChrisChasm this function belongs in the user class.

**Comment by corsacca** [comment]
getting this error: 
image

File: dt-users/users-template.php [hunk 2]
```diff
@@ -257,68 +285,88 @@ function dt_build_user_fields_display( array $usermeta ): array
...
+    
+    $team_members_list = [];
+    
+    $teams = wp_get_object_terms( $user_id, 'user-group' );
+    if( empty( $teams ) ) {
```
Comment (corsacca) [inline dt-users/users-template.php]: wp_get_object_terms throws an error. This code needs to catch it.

File: dt-users/users-template.php (hunk 1 above)
Comment (ChrisChasm) [reply dt-users/users-template.php]: Good point. Actually, it looks like I've drifted a number of functions over to this file that I should probably have located in the user class. I'll do some refactoring.

File: dt-users/users-template.php (hunk 2 above)
Comment (ChrisChasm) [reply dt-users/users-template.php]: Yep. I see that now. Thanks.

Comment (ChrisChasm) [reply dt-users/users-template.php]: Fixed.

**Comment by ChrisChasm** [comment]
314c59b05eac171f6aac0e124f223b2bbe7b648f

**Pull request merged**